app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/NavigationOverlayFragment.kt

```diff
@@ -282,16 +295,15 @@ class NavigationOverlayFragment : Fragment() {
         megaTileTryAgainButton.setOnClickListener { _ ->
             pocketViewModel.update()
             initMegaTile()
-            updateFocusableViews()
-            pocketVideoMegaTileView.requestFocus()
         }
-        updateFocusableViews()
     }
 
     private fun hideMegaTileError() {
         pocketVideosContainer.visibility = View.VISIBLE
         pocketErrorContainer.visibility = View.GONE
-        updateFocusableViews()
+
+        // View.focusable = INT only available Android API > 26 :(
+        pocketVideoMegaTileView.setFocusable(true)
```

Comment:
What is `View.focusable = INT`?

Reply:
View.focusable takes in Int type only, not boolean (unless I'm mistaken)